sensor/upgrader/plan/preserve_test.go

```diff
@@ -152,7 +153,7 @@ func TestPreserveResources(t *testing.T) {
 	var mergedDS v1.DaemonSet
 	require.NoError(t, convert(scheme.Scheme, newDSUnstructured, &mergedDS))
 
-	assert.Equal(t, expectedMergedDS, &mergedDS)
+	protoassert.Equal(t, expectedMergedDS, &mergedDS)
```

Comment:
The objects here are from package `k8s.io/api/apps/v1`. They probably don't need protoassert comparison yet.